src/live_restore/live_restore_fs.c

```diff
@@ -1168,6 +1233,7 @@ __wt_os_live_restore_fs(
     WT_ERR(__validate_live_restore_path(lr_fs->os_file_system, session, lr_fs->source.home));
 
     lr_fs->source.which = WT_LIVE_RESTORE_FS_LAYER_SOURCE;
+    __live_restore_get_fs_block_size(&lr_fs->source);
```

Comment:
We don't use the source layers block size, so I guess we could move `fs_block_size` up into the `WT_LIVE_RESTORE_FS` struct? Open to inputs here